admin_panel/src/commonMain/kotlin/net/thechance/mena/admin_panel/presentation/screen/dukan_managements/component/DukanManagementHeader.kt

```diff
@@ -0,0 +1,86 @@
+package net.thechance.mena.admin_panel.presentation.screen.dukan_managements.component
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.widthIn
+import androidx.compose.foundation.layout.wrapContentHeight
+import androidx.compose.foundation.layout.wrapContentSize
+import androidx.compose.foundation.shape.CircleShape
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.unit.dp
+import net.thechance.mena.admin_panel.presentation.component.SearchBar
+import net.thechance.mena.admin_panel.resources.Res
+import net.thechance.mena.admin_panel.resources.dukans
+import net.thechance.mena.admin_panel.resources.search_hint
+import net.thechance.mena.designsystem.presentation.component.text.Text
+import net.thechance.mena.designsystem.presentation.theme.theme.Theme
+import org.jetbrains.compose.resources.stringResource
+
+@Composable
+fun DukanManagementHeader(
+    dukansNumbers: Int,
+    onQueryChange: (String) -> Unit,
+    onClearQueryClicked: () -> Unit,
+    query: String,
+    modifier: Modifier = Modifier
+) {
+    Box(
+        modifier = modifier
+            .padding(start = 16.dp, bottom = 12.dp)
+            .fillMaxWidth()
+            .wrapContentHeight(),
+    ) {
+        DukansCount(
+            dukansNumbers = dukansNumbers.toString(),
+            modifier = Modifier.align(Alignment.CenterStart)
+        )
```

Comment:
I think it is TopStart
Compare between the header here and the one in the dukan requests 
```suggestion
        DukansCount(
            dukansNumbers = dukansNumbers.toString(),
            modifier = Modifier.align(Alignment.TopStart)
        )
```

https://github.com/user-attachments/assets/7f71f55b-a485-46bd-b33d-f629ad125bc7

Reply:
I thought that it should be aligned with search bar